### PR TITLE
Feature 37 Implement environment layer, add OIDC configuration, other cleanup

### LINKED ISSUE #37: Add a configuration engine
Since the WebUI will be used by several rucio communities, its UI needs to be customizable based on environment variables and/or configuration files provided during its deployment.
At present, the  [`rucio.cfg`](https://github.com/rucio/rucio/blob/a5b3706fdf6085fb0d80ad697357972a6edc5d74/etc/rucio.cfg.template#L237-L249) and [environment variables](https://github.com/rucio/containers/tree/master/ui) are used to configure the old UI. 
In most Kubernetes-based deployments, the configuration is supplied as environment variables and then a rucio.cfg is generated at a later stage.

For the new UI, since it is independent of rucio server and rucio core, we would not require a section in `rucio.cfg`, and can just use environment variables directly to configure the app.

To begin with, the oidc config variables specified in [rucio.cfg](https://github.com/rucio/rucio/blob/a5b3706fdf6085fb0d80ad697357972a6edc5d74/etc/rucio.cfg.template#L237-L249) must be made available to the app via environment variables.

### PR DESCRIPTION
**Pull request by nimishbongale**
Closes #37 

1. Utilizes the dotenv packages present out of the box from create-react-app
2. Specifies a `.env` file for local environment variables
3. Adds a helper utility function that serves us the environment variables
4. If the flag "REACT_APP_local" is set to true, then all local environment variables will get picked from the .env file, but otherwise all environment variables set in the deployment containers will get picked up.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,15 @@
+REACT_APP_local = true
```
Comment (maany) [inline .env:1]: move to .env.template
and add a section in the README.md indicating the developer should create his own.

File: src/util.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// access any environment variable with env('KEY')
+export const env = (value: string): string | undefined => {
```
Comment (maany) [inline src/util.ts:2]: File can be moved to tools folder and add a section in README.md telling how to use this ;)

**Review by maany — CHANGES_REQUESTED** [review]
Looks good. Minor changes required

File: .env (hunk 1 above)
Comment (nimishbongale) [reply .env]: Done!

File: src/util.ts (hunk 2 above)
Comment (nimishbongale) [reply src/util.ts]: Done!

**Pull request merged**